dev-packages/application-package/src/application-props.ts

```diff
@@ -113,12 +113,24 @@ export interface FrontendApplicationConfig extends ApplicationConfig {
      */
     readonly applicationName: string;
 
+    /**
+     * Electron specific configuration.
+     */
+    readonly electron?: Readonly<ElectronFrontendApplicationConfig>;
+}
+
+export interface ElectronFrontendApplicationConfig extends ApplicationConfig {
```

Comment:
I don't think it should extend `ApplicationConfig`

Reply:
done